generators/app/templates/src/app/screens.ejs

```diff
@@ -61,81 +50,38 @@ const InitialLoadingScreenContainer = connect(loadingScreenMapStateToProps)(Init
 // ------------------ Initial loading screen end
 
 <%_ } _%>
-export default StackNavigator({
-  <%_ if (features.login) { _%>
-  [Routes.InitialLoading]: { screen: InitialLoadingScreenContainer },
-  [Routes.Login]: {
-    screen: Login,
-    navigationOptions: {
-      title: 'Login'
-    }
-  },
-  <%_ if (features.tabs) { _%>
-  [Routes.Home]: {
-    screen: TabNavigator({
-      [Routes.Tab1]: {
-        screen: Home,
-        navigationOptions: { title: 'Tab 1' }
-      },
-      [Routes.Tab2]: {
-        screen: Home,
-        navigationOptions: { title: 'Tab 2' }
-      }
-    }),
-    navigationOptions: (<%_ if (features.drawerandroid || features.drawerios) { _%>{ navigation }<%_ } _%>) => ({
-      <%_ if (features.drawerandroid || features.drawerios) { _%>
-      headerRight: (
-        <Button onPress={() => navigation.dispatch(drawerActions.drawerToggled())} title="Drawer" />
-      ),
-      <%_ } _%>
-      title: 'Home'
-    })
-  }
-  <%_ } else { _%>
-  [Routes.Home]: {
-    screen: Home,
-    navigationOptions: (<%_ if (features.drawerandroid || features.drawerios) { _%>{ navigation }<%_ } _%>) => ({
-      <%_ if (features.drawerandroid || features.drawerios) { _%>
-      headerRight: (
-        <Button onPress={() => navigation.dispatch(drawerActions.drawerToggled())} title="Drawer" />
+export default StackNavigator(
+  {
+    <%_ if (features.login) { _%>
+    [Routes.InitialLoading]: {
+      screen: InitialLoadingScreenContainer,
+      navigationOptions: screensNavOptions[Routes.InitialLoading]
+    },
```

Comment:
This pattern is being repeated over projects:
```
[Routes.Component]: {
  screen: Component,
  navigationOptions: screensNavOptions[Routes.Component]
}
```
It would be great to extract this over the screens being declared. What do you think ?

Reply:
Yep, i think so. We should think about a recursive pattern here due to the tabs or nested tabs.

Reply:
@mvbattan I'm going to add a card to solve this later, since I'm with little time to solve this nice to have, so we can merge the pr now